beater/config/kibana.go

```diff
@@ -42,7 +42,7 @@ func (k *KibanaConfig) Unpack(cfg *common.Config) error {
 
 func defaultKibanaConfig() KibanaConfig {
 	return KibanaConfig{
-		Enabled:      false,
+		Enabled:      true,
```

Comment:
Does it make sense to enable Kibana config in all cases? It may not always be available. Under Fleet, is it _ever_ available?

Reply:
My understanding was that we communicate less / not at all with kibana, but the stackpack was enabling it for major version > 7: https://github.com/elastic/cloud-assets/blob/master/stackpack/apm/config/apm-server.yml#L121-L126

Reply:
I think we'll want kibana enabled in 8.0 for situations where users have upgraded from a previous version, and still running standalone. In that case, we'll still want the Kibana connection for agent config and source maps.

For managed mode, this config file is irrelevant.